src/main/java/world/bentobox/bentobox/schems/Clipboard.java

```diff
@@ -188,14 +197,44 @@ public void pasteIsland(World world, Island island, Runnable task) {
         Location loc = island.getCenter().toVector().subtract(off).toLocation(world);
         // Paste
         if (blockConfig.contains(BLOCK)) {
-            blockConfig.getConfigurationSection(BLOCK).getKeys(false).forEach(b -> pasteBlock(world, island, loc, blockConfig.getConfigurationSection(BLOCK + "." + b)));
+            Iterator<String> it = blockConfig.getConfigurationSection(BLOCK).getKeys(false).iterator();
+            Iterator<String> attachmentsIt = blockConfig.contains(ATTACHED) ? blockConfig.getConfigurationSection(ATTACHED).getKeys(false).iterator() : null;
+            pastingTask = Bukkit.getScheduler().runTaskTimer(plugin, () -> {
```

Comment:
I intend to add another Iterator for Entities. Right now they are included in the blocks, but should have their own YAML section in the schem file.